src/table.rs

```diff
@@ -202,6 +202,19 @@ impl Table {
     pub fn decor(&self, key: &str) -> Option<&Decor> {
         self.items.get(key).map(|kv| &kv.key.decor)
     }
+
+    /// Sets the position of the `Table` within the `Document`.
+    ///
+    /// This will never create invalid toml but, formatting may not be
+    /// preserved (see `tests::test_edit::test_multiple_max_usize_positions`).
```

Comment:
We should mention this method only affects `Document::to_string_in_original_order()`.

Also see my comment about formatting below.